### PR TITLE
Updated the metadata  

### PR DESCRIPTION
**Pull request by Aditi8007**
#Canvas-android
#distributed-ranges 
#oneapi-ci

### CODE REVIEW

File: projects/project-metadata.json [hunk 1]
```diff
@@ -2470,7 +2470,13 @@
           }
         ]
       },
-      "license": null,
+      "license": {
+        "key": "other",
```
Comment (harimohanr) [inline projects/project-metadata.json:2474]: @Aditi8007 As per the readme page of this project, it is licensed under GPLv3. Hence you please add the GPLv3 license information here

File: projects/project-metadata.json [hunk 2]
```diff
@@ -5855,7 +5861,13 @@
           }
         ]
       },
-      "license": null,
+      "license": {
```
Comment (harimohanr) [inline projects/project-metadata.json:5864]: @Aditi8007 As per the project readme, it is licensed under [oneAPI EULA](https://software.intel.com/content/www/us/en/develop/articles/end-user-license-agreement.html). Please update this information in the metadata

File: projects/project-metadata.json [hunk 3]
```diff
@@ -6076,7 +6088,13 @@
...
```
Comment (harimohanr) [inline projects/project-metadata.json:6091]: @Aditi8007 As per the repository LICENSE folder, this project uses a BSD-3-Clause open source license. Please update that information in the license metadata section

Comment (Aditi8007) [reply projects/project-metadata.json:6091]: ok I will update

**Review by harimohanr — CHANGES_REQUESTED** [review]
@Aditi8007 Please check the review comments

File: projects/project-metadata.json [hunk 4]
```diff
@@ -2471,11 +2471,11 @@
         ]
       },
       "license": {
-        "key": "other",
-        "name": "Other",
-        "spdx_id": "NOASSERTION",
-        "url": null,
-        "node_id": "MDc6TGljZW5zZTA="
+        "key": "bsd-3-clause",
+        "name": "BSD 3-Clause \"New\" or \"Revised\" License",
```
Comment (harimohanr) [inline projects/project-metadata.json:2475]: @Aditi8007 the license name is just BSD 3-Clause. remove the remaining text

**Review by harimohanr — APPROVED** [review]
@Aditi8007 Looks okay now

**Pull request merged**